members/admin/admin_actions.py

```diff
@@ -0,0 +1,282 @@
+from datetime import timedelta
+from dateutil.relativedelta import relativedelta
+from django import forms
+from django.contrib import admin
+from django.contrib import messages
+
+from django.contrib.admin.widgets import AdminDateWidget
+from django.db import transaction
+from django.utils import timezone
+from django.utils.safestring import mark_safe
+from django.utils.html import escape
+from django.shortcuts import render
+
+from members.models import (
+    Activity,
+    ActivityInvite,
+    ActivityParticipant,
+    Department,
+    Person,
+    WaitingList,
+)
+
+
+class AdminActions(admin.ModelAdmin):
+    def invite_many_to_activity_common_action(modelAdmin, request, queryset):
```

Comment:
Skal vi ikke bare kalde den for
```suggestion
    def invite_many_to_activity_action(modelAdmin, request, queryset):
```
eller er der en særlig grund til `common` i navnet?

Reply:
common_ fjernet fra invite_many_to_activity_common_action (søg-og-erstat i alle filer)